nsqlookupd/lookup_protocol_v1.go

```diff
@@ -45,8 +45,8 @@ func (p *LookupProtocolV1) IOLoop(conn net.Conn) error {
 			}
 			p.ctx.nsqlookupd.logf("ERROR: [%s] - %s%s", client, err, ctx)
 
-			_, err = protocol.SendResponse(client, []byte(err.Error()))
-			if err != nil {
+			_, sendErr := protocol.SendResponse(client, []byte(err.Error()))
```

Comment:
It seems like this error is now "lost" (i.e. it's not even logged).  Should we at least log it?